plugins/MachineSettingsAction/MachineSettingsAction.qml

```diff
@@ -164,7 +164,7 @@ Cura.MachineAction
                                 id: gcodeFlavorComboBox
                                 sourceComponent: comboBoxWithOptions
                                 property string settingKey: "machine_gcode_flavor"
-                                property string label: catalog.i18nc("@label", "GCode flavor")
+                                property string label: catalog.i18nc("@label", "Gcode flavor")
```

Comment:
It seems not to be very standardized. :cry: I personally prefer to use "G-Code".

Using current state of master branch (only the Cura repo) I see the following. This searches EVERYTHING including non-user visible text.

grep -r "GCode" | wc -l
495

grep -r "G-Code" | wc -l
184

grep -r "G-code" | wc -l
175

grep -r "gcode" | wc -l
594

grep -r "g-code" | wc -l
139